components/script/dom/bindings/str.rs

```diff
@@ -28,10 +29,21 @@ impl ByteString {
         str::from_utf8(&vec).ok()
     }
 
+    /// Returns ownership of the underlying Vec<u8> the ByteString is
+    /// unusable afterwards as it has been moved
```

Comment:
`Takes ownership of the underlying Vec<u8>. The ByteString is unusable afterwards.`